src/code/z_effect_soft_sprite.c

```diff
@@ -155,12 +156,16 @@ s32 EffectSs_FindSlot(s32 priority, s32* pIndex) {
 void EffectSs_Insert(PlayState* play, EffectSs* effectSs) {
     s32 index;
 
+#ifdef ENABLE_FRAMERATE_OPTIONS
     if (FrameAdvance_IsEnabled(play) != true) {
```

Comment:
We could probably just create a macro for this check here. For example
```
if (FRAME_ADVANCE_DISABLED()) {
    ...
}
```
In the debug code path this would of course be defined as the same check here, but otherwise would just simply be defined as true.
This also goes for some other things in this PR I think, so it is worth considering creating other macros like this for other debug specific checks.

Reply:
Now that I am thinking about this more, we could just simply make a generic `DBG_COND()` macro, which takes the condition as it's input, so then we could use it everywhere.